components/x-teaser/src/Image.jsx

```diff
@@ -25,26 +25,30 @@ const LazyImage = ({ src, lazyLoad }) => {
 }
 
 export default ({ relativeUrl, url, image, imageSize, imageLazyLoad, imageHighestQuality, ...props }) => {
-	const displayUrl = relativeUrl || url
-	const useImageService = !(image.url.startsWith('data:') || image.url.startsWith('blob:'))
-	const options = imageSize === 'XXL' && imageHighestQuality ? { quality: 'highest' } : {}
-	const imageSrc = useImageService ? imageService(image.url, ImageSizes[imageSize], options) : image.url
-	const ImageComponent = imageLazyLoad ? LazyImage : NormalImage
+	if (!image || (image && !image.url)) {
+		return null
+	} else {
```

Comment:
I think we can remove this `else`

```js
if (!image || (image && !image.url)) {
  return null
}

const displayUrl = relativeUrl || url
...
```

Reply:
I think less nesting would be better. In case when we need to add other conditions in the future.